docs/installation/in-cluster/eks/_index.md

```diff
@@ -0,0 +1,143 @@
+![Specify Networking](./cluster/specify_networking.png)
+7. Click next to go to the Configure Observability view, For this tutorial we will use the default settings.
+![Configure Observability](./cluster/configure_observability.png)
+8. Click next to go to the Select Add-ons view, For this tutorial we will use the default selected add-ons.
+![Select Add-ons](./cluster/select_addons.png)
+9. Click next to go to the Configure selected add-ons view, For this tutorial we will use the default settings.
+![Configure Add-ons](./cluster/configure_addons.png)
+10. Click next to go to the Review and Create view where you can review all the settings and create the cluster.
+
+## Configuring OIDC provider for EKS
+After creating the EKS cluster, we need to configure the OIDC provider for the cluster. This will allow the cluster to authenticate users using Cognito.
+1. Go to the access tab on the cluster overview page of the cluster you created above and click on the Associate OIDC Identity provider.
+2. In the Associate OIDC Identity provider view, Enter a name for the Identity Provider Configuration, enter the issuer URL which can be found in the cognito user pool view you created. It should be something like https://cognito-idp.<YOUR_REGION_HERE>.amazonaws.com/USER_POOL_ID, Client ID can be found in the app integration tab on the user pool overview page. The other fields can be filled as shown in the image below.
+![Associate OIDC Identity provider](./cluster/associate_oidc_identity_provider.png)
+
+## Fetching EKS cluster locally to deploy Headlamp on it
+Make Sure you have the aws cli installed and setup with the necessary permissions to interact with the EKS cluster.
+1. Installation steps for aws cli can be found [here](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+2. Signin to the aws cli can be done by following the steps [here](https://docs.aws.amazon.com/signin/latest/userguide/command-line-sign-in.html)
+
+After setting up the aws cli, you can fetch the kubeconfig for the EKS cluster by running the following command
+```bash
+aws eks --region <YOUR_REGION_HERE> update-kubeconfig --name <YOUR_CLUSTER_NAME_HERE>
+```
+Where <YOUR_REGION_HERE> is the region where the EKS cluster is created and <YOUR_CLUSTER_NAME_HERE> is the name of the EKS cluster.
+
+## Deploying Headlamp on the EKS cluster
+1. First make sure you have helm installed on your local machine, If not you can follow the installation steps [here](https://helm.sh/docs/intro/install/)
+2. Once your cluster is operational, you need to configure a cluster role and a cluster role binding for the cognito user (You can create a cognito user from the aws console or through the sign up page which shows up when headlamp starts the login dance). This step is essential for enabling the Kubernetes API server to identify the user. In this example, we'll be associating the user with the predefined `cluster-admin` Role.
```

Comment:
I tried to make this a bit easier to read...

---

2. Your cluster should be running. 

3. Create a Cognito user through the AWS console. You will need the email for the next step.

4. Inside Kubernetes we need to give users a "cluster-admin" [ClusterRole](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-and-clusterrole). We do this by setting up a ClusterRoleBinding. So the cluster can authorize these users.

... example


5. Create a `values.yaml` file...

Reply:
updated